apps/vscode/vscode.win.talon

```diff
@@ -0,0 +1,235 @@
+# Microsoft - Visual Studio Code
+# see app/vscode.talon for voice commands
+
+os: windows
+app: Visual Studio Code
+app: Code.exe
+-
+
+
+tag(): git
+# TODO When tags are supported
+# tag(): ide
+
+# General
+action(user.ide_command_palette):
+  key(ctrl-shift-p)
+
+action(app.tab_next):
+  key(ctrl-k)
+  key(ctrl-pagedown)
+  
+action(app.tab_previous):
+  key(ctrl-k)
+  key(ctrl-pageup)
+  
+# Folding
+action(user.ide_expand_deep):
+  key(ctrl-k ctrl-])
+action(user.ide_expand_all):
+  key(ctrl-k ctrl-j)
+action(user.ide_expand_region):
+  key(ctrl-shift-])
+action(user.ide_collapse_deep):
+  key(ctrl-k ctrl-[)
+action(user.ide_collapse_all):
+  key(ctrl-k ctrl-0)
+action(user.ide_collapse_region):
+  key(ctrl-shift-[)
+
+# Splits
+action(user.ide_split_right):
+  user.ide_command_palette()
+  insert("workbench.action.splitEditorRight")
+  key(enter)
+
+action(user.ide_split_left):
+  user.ide_command_palette()
+  insert("workbench.action.splitEditorLeft")
+  key(enter)
+
+action(user.ide_split_up):
+  user.ide_command_palette()
+  insert("workbench.action.splitEditorUp")
+  key(enter)
+
+action(user.ide_split_down):
+  user.ide_command_palette()
+  insert("workbench.action.splitEditorDown")
+  key(enter)
+
+#Refactor
+
+action(user.ide_refactor):
+  key(ctrl-a)
+  # Format Document
+  key(alt-shift-f)
```

Comment:
This is looks like the equivalent of the `ide_fix_format.` I think this should triggering the refactor command ('ctrl-shift-r'), no?.